.semaphore/semaphore.yml

```diff
@@ -0,0 +1,89 @@
+version: v1.0
+name: Kube Controllers
+
+execution_time_limit:
+  hours: 2
```

Comment:
same here - looks like typical job is under 15 minutes, maybe set to `minutes: 30` for e.g.?

Reply:
Ya good point, I'll change it.